src/main/java/build/buf/protovalidate/internal/evaluator/EvaluatorBuilder.java

```diff
@@ -27,12 +27,9 @@
 import build.buf.validate.MessageConstraints;
 import build.buf.validate.OneofConstraints;
 import build.buf.validate.ValidateProto;
-import com.google.protobuf.Descriptors;
+import com.google.protobuf.*;
```

Comment:
I think we should update our code style to prevent wildcard imports for consistency across projects.